website/blog/2021-11-23-how-to-upgrade-dbt-versions.md

```diff
@@ -97,7 +99,7 @@ require-dbt-version: ">=0.17.2"
 ...
 ```
 
-You can add an upper bound of supported versions like this: `[">=0.20.0", "<=1.0.0"]`, but for an internal analytics project it's probably overkill. Fun fact: this upper bound is how package vendors stop users from accidentally using an old version of a package like dbt-utils - more on this in a bit!
+You can add an upper bound of supported versions like this: `[">=1.4.0", "<=1.5.0"]`, but for an internal analytics project it's probably overkill. Fun fact: this upper bound is how package vendors stop users from accidentally using an old version of a package like dbt-utils - more on this in a bit!
```

Comment:
This doesn't make sense in the context of the old blog either

Reply:
Blast, this was a mistake. Will revert the blog to it's original format. Was only supposed to impact /docs and /ref pages. Apoloogies